Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG commit=develop
+FROM elifesciences/peerscout_client:${commit} AS client
+FROM elifesciences/python:cacd250ec201feab491c0738f261561b5360997b
+
+USER elife
+ENV PROJECT_FOLDER=/srv/peerscout
+RUN mkdir ${PROJECT_FOLDER}
+WORKDIR ${PROJECT_FOLDER}
+COPY --chown=elife:elife install.sh requirements.txt requirements-debug.txt ${PROJECT_FOLDER}/
+RUN /bin/bash install.sh
+
+COPY --from=client --chown=elife:elife /home/node/client/ ${PROJECT_FOLDER}/client/
+COPY --chown=elife:elife docvec_model/ ${PROJECT_FOLDER}/docvec_model/
+COPY --chown=elife:elife peerscout/ ${PROJECT_FOLDER}/peerscout/
+COPY --chown=elife:elife server.sh ${PROJECT_FOLDER}/
+COPY --chown=elife:elife project_tests.sh ${PROJECT_FOLDER}/
+COPY --chown=elife:elife update-data-and-reload.sh ${PROJECT_FOLDER}/
+COPY --chown=elife:elife migrate-schema.sh ${PROJECT_FOLDER}/
+COPY --chown=elife:elife app-defaults.cfg ${PROJECT_FOLDER}/
+
+
+USER root
+RUN mkdir .data && chown www-data:www-data .data
+RUN mkdir logs && chown www-data:www-data logs
+
+# Test
```

Comment:
these should definitely go in `Dockerfile.ci`

Reply:
But we would still need those files. In production we would map them to volumes I guess. But wouldn't we still create them if they don't exist?

Reply:
I shouldn't have replied to the message on the main overview page which was showing the lines above. You meant the lines below I presume..

Reply:
About `.data` and `logs`: yes, these are defaults and would be mounted by volumes so that they persist. Do they need any default content on a new instance?

Reply:
On the files below: yes, that part is ready to be moved in `.ci`